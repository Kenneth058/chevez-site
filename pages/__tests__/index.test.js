import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'
import renderer from 'react-test-renderer'
import Index from '../index'

test('Should render index page', () => {
  render(<Index />)
  expect(screen.getByTestId('test-1')).toBeInTheDocument()
})

test('Should matches snapshot', () => {
  const tree = renderer.create(<Index />)
  expect(tree).toMatchSnapshot()
})
