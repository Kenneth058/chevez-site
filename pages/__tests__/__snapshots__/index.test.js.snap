// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should matches snapshot 1`] = `
<h1
  className="bg-black text-white text-lg"
  data-testid="test-1"
>
  Hola mundo a mi canal
</h1>
`;
